vmpooler-provider-vsphere.gemspec

```
@@ -17,7 +17,7 @@ Gem::Specification.new do |s|
   s.require_paths = ["lib"]
   s.add_dependency 'rbvmomi', '>= 2.1', '< 4.0'
   
-  s.add_development_dependency 'vmpooler', '~> 1.3', '>= 1.3.0'
+  s.add_development_dependency 'vmpooler', '~> 2', '>= 2.1' # renaming done in version 2
```

```suggestion
  s.add_development_dependency 'vmpooler', '~> 2.0', '>= 2.1' # renaming done in version 2
```
I am fairly sure that without the `.0` it would be able to bump the major version too.

---

you are right,
changed to ~> 2.1 since this is equivalent to 
'>= 2.1.0', '< 3.0'